build.html

```diff
@@ -0,0 +1,80 @@
+<div class="is-loading" data-interactive-floorplan-id="{{ id }}">
```

Comment:
How about `{{ uuid }}`. Don't we usually add that too? `uuid` is consistent across master and production apps.